manual/cli.rst

```diff
@@ -1,4 +1,100 @@
+| -q N       | --number-of-plates=N | number of plates                                                                                                              |
++------------+----------------------+-------------------------------------------------------------------------------------------------------------------------------+
+|            | --recursion-limit=N  | you need that just if you encounter an error while generating very large maps                                                 |
++------------+----------------------+-------------------------------------------------------------------------------------------------------------------------------+
+| -v         | --verbose            | Enable verbose messages                                                                                                       |
++------------+----------------------+-------------------------------------------------------------------------------------------------------------------------------+
+|            | --version            | Display version information                                                                                                   |
++------------+----------------------+-------------------------------------------------------------------------------------------------------------------------------+
+| --bw       | --black-and-white    | Draw maps in black and white                                                                                                  |
++------------+----------------------+-------------------------------------------------------------------------------------------------------------------------------+
+
+
+Options valid only for generate
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+| Short     | Long                       | Description                                                                                          |
++===========+============================+======================================================================================================+
+| -r FILE   | --rivers=FILE              | produce a map of rivers, after the option it expects the name of the file where to generate the map  |
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+| --gs      | --grayscale-heightmap      | produce a grayscale heightmap                                                                        |
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+|           | --ocean_level=N            |  elevation cut off for sea level [default = 1.0]                                                     |
```

Comment:
well, here we document the intended behavior. We will fix it.... sooner or later :)